android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandlerOrchestrator.java

```diff
@@ -321,21 +317,32 @@ private void deliverEventToGestureHandler(GestureHandler handler, MotionEvent ev
     }
   }
 
-  private void extractCoordsForView(View view, MotionEvent event, float[] outputCoords) {
+  /**
+   * extractCoordsForView performs extraction of coords for event in rootView.
+   * In addition it checks whether all of parents for view related to handler are attached.
+   * There might be a rarely observed issue when view has been detached and handler's
+   * state hasn't been change to failed or ended yet. Probably it's a result of some race condition
+   * and stopping delivering for this handler and changing its state to failed of end appear
+   * to be good enough solution.
+   */
+  private boolean extractCoordsForView(View view, MotionEvent event, float[] outputCoords) {
```

Comment:
I prefer that we don't change the signature of this method. It isn't very clear on when it should return what. We should either change its name if we want it to perform an additional test but I think the best would be to do the following:
1) leave this method untouched
2) add a new method that tests if view is attached under `mWrapperView` (e.g. `isViewAttachedUnderWrapper` which traverses the hierarchy up until it reaches wrapper)
3) use the above method directly in `deliverEventToGestureHandler` even before we test `handler.wantEvents`
4) if view is not attached we immediately call `cancel` on the corresponding handler to match the behavior from https://github.com/kmagiera/react-native-gesture-handler/blob/master/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRegistry.java#L65